src/components/staking/Summary.svelte

```diff
@@ -55,16 +55,24 @@
               votingClass = "text-red";
               voteKeyword = "cannot_votes";
             } else {
-              if(oldestValidLock && $stakingData.proposals[0].block.timestamp < Number(oldestValidLock.lockedAt)) {
+              let proposal = get($stakingData.proposals, 0);
```

Comment:
this is not the best way to use get. See next comment

Reply:
ok, noted down for future